config/locales/en.txt

```diff
@@ -858,6 +858,7 @@
   add_objects: Add [TYPES]
   all_objects: All [TYPES]
   cancel_and_show: Cancel (Show [TYPE])
+  cancel_to_index: Cancel ([TYPE] Index)
```

Comment:
```suggestion
  cancel_to_index: "[:CANCEL] ([TYPE] [:INDEX])"
```
Works well to automatically translate for many languages, while still allowing manual translation.